src/game_manager.py

```diff
@@ -294,7 +300,7 @@ def on_key_pressed(self, controller, keyval, keycode, state, row: int, col: int)
 
     def on_number_selected(self, num_button: Gtk.Button, cell: SudokuCell, popover):
         number = num_button.get_label()
-        self._fill_cell(cell, number)
+        self._fill_cell(cell, number, False)
```

Comment:
```suggestion
        self._fill_cell(cell, number)
```
See? You have to add `False` in here, and missing it somewhere could cause unwanted bugs/issues. However, after defaulting ctrl_is_pressed to False, we no longer need to do this.